gcp/iap-ingress/base/config-map.yaml

```diff
@@ -115,7 +115,7 @@ data:
       . /var/shared/healthz.env
 
       # If node port or backend id change, so does the JWT audience.
-      CURR_NODE_PORT=$(kubectl --namespace=${NAMESPACE} get svc ${SERVICE} -o jsonpath='{.spec.ports[0].nodePort}')
+      CURR_NODE_PORT=$(kubectl --namespace=${NAMESPACE} get svc ${SERVICE} -o jsonpath='{.spec.ports[?(@.name=="'${PORT_NAME}'")].nodePort}')
```

Comment:
is PORT_NAME set? Should we just use http2 like other places:
https://github.com/kubeflow/manifests/blob/master/gcp/iap-ingress/base/config-map.yaml#L73

Reply:
thanks, I didn't look closely enough at the structure of this file. updated